pkg/iputils/iputil.go

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
I think we should avoid `utils` if we can as is recommended by  effective Go. I suggest just `ip` (or `net` altho that would require import aliasing if also importing Go `net`).

Reply:
How about a more specific package, `cidr`, for CIDR-related functions? I’ve been meaning to rework the CIDR allocation anyway at some point (there’s a nice package somewhere that helps with all that).

Reply:
yeah  something meaningfully specific w/o `util`. I like `cidr` - then the function  would become `cidr.IsOverlapping(...)`.

Reply:
Perhaps we should also put this in `admiral`.

Reply:
We briefly discussed (over slack) whether to put this API in admiral, but since it's used only by Operator (apart from Submariner) created a different package in the Submariner repo itself.